core-jdk8/src/main/java/org/mapstruct/Mapping.java

```diff
@@ -136,4 +136,11 @@
      * @return the qualifiers
      */
     Class<? extends Annotation>[] qualifiedBy() default { };
+
+   /**
+     * Specifies the result type of the mapping method to be used in case mapping factory methods qualify.
```

Comment:
How do factory methods fit into the picture here? Should it be

"Specifieds the result type of the property mapping method to be used in case several methods qualify"

?